controllers/verticalscalingpolicy_controller.go

```diff
@@ -351,14 +352,14 @@ func instanceFamilyExists(family string, instanceTypesInfo []*ec2.InstanceTypeIn
 	return true
 }
 
-func (r *VerticalScalingPolicyReconciler) NotifyTargets(vsp *v1alpha1.VerticalScalingPolicy) {
-	targets := vsp.Spec.Targets
+func (r *VerticalScalingPolicyReconciler) NotifyTargets(targets []string) {
 	for _, target := range targets {
 		r.ManagerContext.InstanceGroupEvents <- event.GenericEvent{
 			Object: &metav1.PartialObjectMetadata{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:      target.Name,
-					Namespace: target.Namespace,
+					// Target name is in format my-namespace/my-ig
+					Namespace: strings.Split(target, "/")[0],
+					Name:      strings.Split(target, "/")[1],
```

Comment:
In future, we can use regex match to get namespace and ig names from target string to avoid array out of bounds exception

Reply:
Yeah, I will add a TODO note here